src/Symfony/Component/Messenger/Middleware/StackMiddleware.php

```diff
@@ -65,16 +65,16 @@ public function handle(Envelope $envelope, StackInterface $stack): Envelope
  */
 class MiddlewareStack
 {
-    public $iterator;
-    public $stack = [];
+    public \Iterator $iterator;
```

Comment:
Line 84 will break: `$this->iterator = null`

Reply:
fixed